src/internal/operations/backup.go

```diff
@@ -93,9 +93,15 @@ func (op *BackupOperation) Run(ctx context.Context) (err error) {
 	ctx, end := D.Span(ctx, "operations:backup:run")
 	defer end()
 
+	m365, err := op.account.M365Config()
```

Comment:
You can retrieve the ID without the m365 configuration with `op.account.ID()`.  We should probably rename that func to `TenantID()`.  But, hey, at least it's there.